highfidelity/hifi
# Hand Azimuth improvement

This pr puts the hand azimuth computation in shoulder space.  That means that we can now sit down with hmd lean recenter off and have the hands behind the origin of MyAvatar without messing up the azimuth computation.  I think it also look slightly better when walking and leaning sideways.

### Test Plan:
1. open interface and go to vr mode.
2. open settings/controls/Avatar Lean Behavior and set it to Disabled
3. Sit down on the floor or a chair in real life.  Your avatar knees should bend and you will be sitting behind the location of your feet.
4. rotate your arms to the right and left and verify that the torso follows the arms

https://highfidelity.manuscript.com/f/cases/19436/Bug-fix-for-shoulder-turning-with-HMD-lean-recenter-off
